tests/regress/emu_clear_errors.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import binascii
+import regress
+
+from unicorn import *
+from unicorn.x86_const import *
+
+
+CODE = binascii.unhexlify(b"".join([
+    b"8B 74 01 28",       # mov esi, dword ptr [ecx + eax + 0x28]  mapped: 0x1000
+    b"03 F0",             # add esi, eax                                   0x1004
+    b"8D 45 FC",          # lea eax, dword ptr [ebp - 4]                   0x1006
+    b"50",                # push eax                                       0x1009
+    b"6A 40",             # push 0x40                                      0x100A
+    b"6A 10",             # push 0x10                                      0x100C
+    b"56",                # push esi                                       0x100E
+    b"FF 15 20 20 00 10"  # call some address                              0x100F
+  ]).replace(" ", ""))
+
+
+def showpc(mu):
+    pc = mu.reg_read(UC_X86_REG_EIP)
+    print("pc: 0x%x" % (pc))
+
+
+class HookCodeStopEmuTest(regress.RegressTest):
+    def test_hook_code_stop_emu(self):
+        mu = Uc(UC_ARCH_X86, UC_MODE_32)
+
+        # base of CODE
+        mu.mem_map(0x1000, 0x1000)
+        mu.mem_write(0x1000, CODE)
+        mu.reg_write(UC_X86_REG_EIP, 0x1000)
```

Comment:
this is not important, but there is no need to set EIP like this. below when you call `mu.emu_start()`, EIP is automatically set at 0x1000 (the first argument)